core/src/main/java/hudson/node_monitors/ResponseTimeMonitor.java

```diff
@@ -44,8 +47,22 @@
  * @author Kohsuke Kawaguchi
  */
 public class ResponseTimeMonitor extends NodeMonitor {
+
+    @DataBoundConstructor
+    public ResponseTimeMonitor() {
+    }
+
+    @SuppressFBWarnings(value = "MS_PKGPROTECT", justification = "for backward compatibility")
+    public static /*almost final*/ AbstractNodeMonitorDescriptor<Data> DESCRIPTOR;
+
     @Extension
-    public static final AbstractNodeMonitorDescriptor<Data> DESCRIPTOR = new AbstractAsyncNodeMonitorDescriptor<>() {
+    @Symbol("responseTime")
+    public static class DescriptorImpl extends AbstractAsyncNodeMonitorDescriptor<Data> {
```

Comment:
Unsure whether we want `@since` for this. Technically, yes?

Reply:
dito